src/views/Workflow.vue

```diff
@@ -152,6 +177,7 @@ export default {
   beforeRouteLeave (to, from, next) {
     this.$workflowService.stopDeltasSubscription()
     this.tree.clear()
+    this.table.clear()
```

Comment:
See comment above.

Reply:
Line replaced with this.table = [] and table appears to be functioning fine following the change.